src/components/utils/sanitizer.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * CodeX Sanitizer
+ *
+ * @module Sanitizer
+ * Clears HTML from taint tags
+ *
+ * @version 2.0.0
+ *
+ * @example
+ *
+ * Sanitizer.clean(yourTaintString, yourConfig);
+ *
+ * {@link SanitizerConfig}
+ */
+
+import * as _ from '../utils';
+
+/**
+ * @typedef {object} SanitizerConfig
+ * @property {object} tags - define tags restrictions
+ *
+ * @example
+ *
+ * tags : {
+ *     p: true,
+ *     a: {
+ *       href: true,
+ *       rel: "nofollow",
+ *       target: "_blank"
+ *     }
+ * }
+ */
+
+import HTMLJanitor from 'html-janitor';
+import { BlockToolData, SanitizerConfig } from '../../../types';
+import { SavedData } from '../../../types/data-formats';
+
+/**
+ * Sanitize Blocks
+ *
+ * Enumerate blocks and clean data
+ *
+ * @param blocksData - blocks' data to sanitize
+ * @param sanitizeConfig — sanitize config to use or function to get config for Tool
+ */
+export function sanitizeBlocks(
```

Comment:
It is the only method that uses the `Block` term and the `SavedData` type. Maybe it is not util? Maybe it should be moved to the BlockManager, for example? What do you think?

Reply:
I think Sanitizer is a still Editor.js only util, so it's ok to leave it here to not overcrowd BlockManager